README.md

```diff
@@ -32,6 +36,18 @@ If you select `ETHERNET` with `UBLOX_ODIN_EVK_W2` you must add this to your `tar
             }
 ```
 
+#### Mbed OS 5.9 and newer
+
+With Mbed OS 5.9, the EMAC SW was refactored and a default network selector is used instead. You must add the following `target-overrides` section to `mbed_app.json`:
+
+```json
+        "UBLOX_EVK_ODIN_W2": {
```

Comment:
This applies to all boards, not just Ublox. All boards can now tell what is the default interface type.

Reply:
All boards that have more than 1 network stack by default? I.e. ethernet + something else. C027, C030, ...

Reply:
Not all, but some. But this is more important on Ethernet usage.

For your configuration file, this applies only Ethernet enabled boards. Ublox ODIN W2 does have WiFi and Ethernet, so you are forced to select a default. Both are basically just Ethernet, from Mbed OS point of view.

In future, Mbed OS 5.10, or earlier, you are able to select between Cellular/WiFi/Ethernet with this same setting.

For 5.9, according to [targets.json](https://github.com/ARMmbed/mbed-os/blob/mbed-os-5.9/targets/targets.json):
Odin W2 default is Wifi (so you need this setting)
C030 default is Ethernet
C027 default is cellular

So if your code does:
```
EthernetInterface eth
```

and board has two ethernet drivers (ODIN_W2) it picks up the default. But if the device has only one ethernet and something else (C030, C027) it picks up the only ethernet driver it has.

Later, once interfaces will be unified (but not ready in 5.9) we are able to do:
```
NetworkInterface *net = NetworkInterface::get_default_instance();
```
And based on `mbed_app.json` it can be anything.